data/united-states/washington.csv

```diff
@@ -236,13 +236,15 @@ e9907162-94c6-4d12-983b-fa8ab96c7e5a,JC Brewhouse,closed,,,,Issaquah,Washington,
 2d416c88-05cf-467a-a144-c056c5d8caeb,Keyhole Valley Brewing,closed,,,,Shelton,Washington,98584,United States,,http://www.keyholevalleybrewing.com,,
 72ceee38-898f-40c2-a1ae-a7cbc3b53aea,Kulshan Brewing Co - K2,micro,1538 Kentucky St,,,Bellingham,Washington,98229-4720,United States,3603895348,http://www.kulshanbrewing.com,-122.4539281,48.75767348
 e0643011-6445-4f77-bdde-052910e1d1ba,Kulshan Brewing Co - Sunnyland,micro,2238 James St,,,Bellingham,Washington,98225-4142,United States,3603895348,http://www.kulshanbrewing.com,-122.4645645,48.7601273
+                                    ,Kulshan Brewing Co - Trackside,location,298 W Laurel St,,,Bellingham,Washington,98225,United States,3603895348,https://kulshanbrewing.com/trackside,-122.485461,48.748621

```

Comment:
Hey @JohnHenry-Ward ! Thanks so much for your contribution! 

Though a space will technically work, the validation script will go green if you delete the space for the ID on this line. Let me know if you want to make that change (for, you know, practice 😄 ), or I can just go ahead and merge anyway since I don't think it'll affect anything during my maintenance workflow. 🍻

Reply:
Ah yeah I was wondering how to resolve that, just pushed the fix! 🍻